docs/developers/fuse-box/flutter-sdk/4-send-batch-txn.mdx

```diff
@@ -0,0 +1,79 @@
+---
+title: Send Batch Transactions
+sidebar_position: 4
+---
+
+### Non-Blocking Transactions
```

Comment:
Non-Blocking Transactions is a feature that can be used in both single & batch transactions.
It needs a separated guide.
You can read about the feature here: https://docs.stackup.sh/docs/useroperation-nonce#why-use-2-dimensional-nonces

Reply:
Moved Non-Blocking to a separate file.